cmd/jag/commands/util.go

```diff
@@ -539,3 +539,20 @@ func getWifiCredentials(cmd *cobra.Command) (string, string, error) {
 	}
 	return wifiSSID, wifiPassword, nil
 }
+
+// isRunningOnBuildbot returns true if the current process is running on a buildbot.
```

Comment:
```suggestion
// Returns true if the current process is running on a buildbot.
```

Reply:
Go convention repeats the function name. (I know...)